utils/docker/Dockerfile.centos.7

```diff
@@ -165,7 +172,10 @@ enabled=1\n\
 gpgcheck=False\n" >> /etc/yum.repos.d/$repo:$branch:$build_number.repo;       \
         done;                                                                 \
         echo "Installing: $QUICKBUILD_DEPS";                                  \
-        echo "$QUICKBUILD_DEPS" | tr '\n' '\0' | xargs -0 yum -y install;     \
+        yum clean all;                                                        \
+        echo "$QUICKBUILD_DEPS" |                                             \
+        tr '\n' '\0' | xargs -0 yum --nogpgcheck -y install;                  \
```

Comment:
Doesn't the repo definition already disable GPG (only) for the repos that it's not usable?  Add that option here is going to disable the GPG check for any upstream dependencies installed with the local packages, and again, we DO NOT want to disable GPG checking for RPMs that we pull from the Internet.

Reply:
Noted above.